### ISSUE TITLE
False error CS0170: Use of possibly unassigned field 'Item1'

### ISSUE THREAD

**Issue by SirDuke** [dotnet/roslyn#16106]
```
public class MyClass
{
    public void Test4(IEnumerable<((int, int), string)> en)
    {
        foreach ((ValueTuple<int,int> t, string s) in en)
        {
            var a =t.Item1; // false error CS0170: Use of possibly unassigned field
        }
    }

    public void Test5(IEnumerable<((int, int), string)> en)
    {
        foreach (((int,int) t, string s) in en)
        {
            var a = t.k; // false error CS0170: Use of possibly unassigned field
        }
    }
}
```

**Comment by alrz** [comment]
The second loop probably should be 
```cs
        foreach (((int ,int k) t , string s)  in en)
```
In which produces CS0170 at `t.k`.
